script/github-actions/check-broken-links.js

```diff
@@ -17,17 +17,17 @@ if (fs.existsSync(reportPath)) {
   const shouldFail =
     brokenLinks.isHomepageBroken ||
     brokenLinks.brokenLinksCount > maxBrokenLinks;
-  const color = shouldFail ? 'danger' : 'warning';
-
+  const color = shouldFail ? '#D33834' : '#FFCC00'; // danger or warning, needs to be in hex
   const heading = `@cmshelpdesk ${brokenLinks.brokenLinksCount} broken links found in the ${envName} build on ${BRANCH_NAME} \n\n${SERVER_URL}\n\n`;
-  const message = `${heading}\n${brokenLinks.summary}`;
+  const slackBlocks = `[{"type": "section","text": {"type": "plain_text","text": "${heading}"}}]`;
```

Comment:
Just to match the spacing of `slackAttachments`...

```suggestion
  const slackBlocks = `[{"type": "section", "text": {"type": "plain_text", "text": "${heading}"}}]`;
```